### DISCUSSION TITLE
Crowdin, translations and translators

### DISCUSSION THREAD

**Discussion by ikanakova** [fast4x/RiMusic#422] [General]
General discussion about translations

**Comment by ikanakova** [comment]
I have a question:
`<string name="show_like_button">Show Like button</string>`
@fast4x The like button is the heart (favorites), right?

**Reply by fast4x** [reply]
yes, in this case is relative to enable it on settings for background player

**Reply by ikanakova** [reply]
OK I'll make some improvements in strings to make it "favorite button" instead of "like button" and other suggestions from crowdin.

**Comment by ikanakova** [comment]
@fast4x I was wondering if it would be possible to post updates in strings.xml a few hours before the release of the new version, so that the translation of the news could be merged into the new version? For example: by a certain hour of a certain day (e.g. to Friday 12:00 AM), the translators would know that they could get the translation done in time for the new version, including the new features.

**Comment by fast4x** [comment]
Okay, I’m trying, but it’s really challenging.
i push now the new version of strings

**Reply by ikanakova** [reply]
If that's the problem, it can stay as it is. I just thought it would be great.

**Reply by fast4x** [reply]
Pushed and synced with crowdin

**Reply by fast4x** [reply]
This version of strings it's optimized, i removed translations not used..

**Comment by comradekingu** [comment]
Can Weblate be used instead of Crowin on account of the latter having problems both in terms of license and utility?

**Reply by fast4x** [reply]
It is a paid service

**Reply by jzitnik-dev** [reply]
> It is a paid service

Yeah, you would need to self host it if you want it for free.

**Reply by comradekingu** [reply]
There is a libre plan for libre software projects https://hosted.weblate.org/hosting/
:)

**Reply by fast4x** [reply]
> There is a libre plan for libre software projects https://hosted.weblate.org/hosting/
> :)

I'll chwck it thanks

**Comment by fast4x** [comment]
Maybe if it's ok localization of RiMusic will migrate to https://gitlocalize.com/

**Comment by fast4x** [comment]
Unfortunately GitLocalize does not support the translation file format used in Android. Migration will not be possible.

**Comment by fast4x** [comment]
Incomplete translations will be eliminated by Crowdin this month.

I'm talking about:
- Afrikaans 
- Arabic
- Esperanto 
- Greek
- Sinhala 
- Telugu 
- Vietnamese 

These languages are translated less than 50% and will be deleted to recover space on Crowdin.

Obviously they will not be deleted from the app.

**Comment by sayyid5416** [comment]
@fast4x Is this `LIBRARY` not available in `app/src/main/res/values-en/strings.xml`?

![image](https://github.com/fast4x/RiMusic/assets/78584556/dd99f09d-2e9e-4a37-949e-bc9d9e2ec370)

**Reply by sayyid5416** [reply]
Also missing for `Player Appearance > PLAYER ACTION BAR > Show Equalizer`

**Reply by fast4x** [reply]
I'll check it thanks

**Comment by fast4x** [comment]
I am pleased to tell you that Crowdin has accepted Rimusic as an opensource project and has granted an opensource license with no limitations compared to the previous free plan.

**Reply by comradekingu** [reply]
The limitations is the project has to be non-commercial.
Crowdin dictating every single condition is also not good.
There are heaps of stuff that applies if you read the terms and conditions and all the spaghetti of surrounding documents, like for example the cookie policy.

**Comment by fast4x** [comment]
As suggested by @JZITNIK-github, I am preparing the website https://rimusic.xyz translation project on Crowdin. This means that the website can also be translated in the same way as RiMusic. I want to clarify that the site has been made multilingual thanks to the implementation of @JZITNIK-github

**Reply by NEVARLeVrai** [reply]
Already translated everything ^^, also the app is becoming very beautiful !

**Comment by fast4x** [comment]
From now it's active a specific url for translation https://translate.rimusic.xyz its same as direct link to Crowdin project page.
